src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/MessageHeaderMapperTest.java

```diff
@@ -70,7 +70,7 @@ void when_mappingMessageToMessageHeader_expect_computedFieldsAreMapped() {
 
         assertAll(
             () -> assertThat(messageHeader.getId()).isEqualTo(MESSAGE_HEADER_ID),
-            () -> assertThat(messageHeader.getTimestamp()).isEqualTo("1970-01-01T00:02:03.000"),
+            () -> assertThat(messageHeader.getTimestamp()).isEqualTo("1970-01-01T01:02:03.000"),
```

Comment:
Maybe I missed something, but should the expected timestamp be unaffected by this PR?